scripts/aur-update.sh

```
@@ -59,6 +71,12 @@ grep -q "sha256sums_x86_64 = ${sha_amd64_fzf}" "$srcinfo" || { echo "AUR update
 grep -q "sha256sums_aarch64 = ${sha_arm64}" "$srcinfo" || { echo "AUR update failed: sha256sums_aarch64 (main)" >&2; exit 1; }
 grep -q "sha256sums_aarch64 = ${sha_arm64_fzf}" "$srcinfo" || { echo "AUR update failed: sha256sums_aarch64 (fzf)" >&2; exit 1; }
 
+# Guard against the issue #141 regression: the download URL must point at the
+# real (unversioned) release asset, not lazy-tmux_<ver>_linux_*.tar.gz (404).
+grep -q "releases/download/v${ver}/lazy-tmux_linux_amd64.tar.gz" "$srcinfo" || { echo "AUR update failed: versioned asset URL in .SRCINFO (issue #141)" >&2; exit 1; }
+grep -qE "releases/download/v${ver}/lazy-tmux_[0-9]" "$srcinfo" && { echo "AUR update failed: .SRCINFO still has a versioned asset URL (issue #141)" >&2; exit 1; }
+grep -q 'releases/download/v${pkgver}/lazy-tmux_linux_amd64.tar.gz' "$pkgbuild" || { echo "AUR update failed: versioned asset URL in PKGBUILD (issue #141)" >&2; exit 1; }
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)